src/main/webapp/add_event_page/add_event.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Add Event</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="initMap()">
+    <div id="header">
+      <h1>Add New Event</h1>
+    </div>
+
+    <div id="form">
+        <form action="/new_event.php">
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="name"><br>
+            <label for="desc">Description:</label>
+            <input type="text" id="desc" name="desc"><br>
```

Comment:
Maybe we could use "textarea" instead of "input" for the description?

Reply:
Yeah I think textarea will be better for the description because it can be multiline input. See more info about textarea [here](https://www.w3schools.com/tags/tag_textarea.asp)